angr/analyses/variable_recovery/engine_base.py

```diff
@@ -562,7 +562,9 @@ def _load(self, richr_addr: RichR, size: int, expr=None):
                 if values:
                     for vs in values.values.values():
                         for v in vs:
-                            for var_offset, var_ in self.state.extract_variables(v):
+                            for _, var_ in self.state.extract_variables(v):
+                                var_: SimStackVariable
```

Comment:
This type annotation is not sound - I observe a SimRegisterVariable in practice.